tests/acceptance/fixtures/responses/universal.js

```diff
@@ -0,0 +1,486 @@
+import { RequestMock } from 'testcafe';
+
+const UniversalSearchResponse = {
+  meta: {
+    uuid: '0180235e-7fcd-5262-247b-d3537eea0002',
+    errors: []
+  },
+  response: {
+    businessId: 3350634,
+    modules: [
+      {
+        verticalConfigId: 'people',
+        resultsCount: 1,
+        encodedState: '',
+        results: [
+          {
+            data: {
+              id: 'Employee-2116',
+              type: 'ce_person',
+              website: 'http://www.test.com',
+              address: {
+                line1: '7900 Westpark Drive',
+                city: 'Mclean',
+                region: 'VA',
+                postalCode: '22102',
+                countryCode: 'US'
+              },
+              associations: [
+                'Runners Association'
+              ],
+              description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Mollis nunc sed id semper risus. Commodo elit at imperdiet dui accumsan sit amet nulla facilisi. Scelerisque viverra mauris in aliquam. A erat nam at lectus. Egestas purus viverra accumsan in nisl nisi scelerisque. Habitant morbi tristique senectus et netus. Congue nisi vitae suscipit tellus mauris a diam maecenas sed.',
+              name: 'Amani Farooque',
+              cityCoordinate: {
+                latitude: 38.936519622802734,
+                longitude: -77.18428039550781
+              },
+              c_allstateLadyCTA: {
+                label: 'Learn More',
+                linkType: 'URL',
+                link: 'http://yext.com'
+              },
+              c_employeeCity: 'Tysons Corner',
+              c_employeeCountry: 'United States',
+              c_employeeDepartment: 'Consulting',
+              c_employeeRegion: 'Virginia',
+              c_employeeTitle: 'Associate Technical Project Manager',
+              c_oliverCta: {
+                url: 'https://www.yext.com/s/2287528/entity/edit2?entityIds=13411251',
+                icon: 'yext',
+                label: 'test cta'
+              },
+              c_startDate: '2017-09-18',
+              displayCoordinate: {
+                latitude: 38.924648,
+                longitude: -77.216859
+              },
+              emails: [
+                'afarooque@yext.com',
+                'af@yext.com',
+                'aplomb@yext.com'
+              ],
+              firstName: 'Amani',
+              geocodedCoordinate: {
+                latitude: 38.924654,
+                longitude: -77.216891
+              },
+              headshot: {
+                url: 'http://a.mktgcdn.com/p/bmc3W88h2mMRXk-wHQ7dB0Em4rR_dfia6OVrAK3LjYU/192x191.png',
```

Comment:
for example, this is a real photo

Reply:
I'm not overly concerned. This is a real photo, but it's Amani's LinkedIn profile picture. Not exactly private info. As for the latitude and longitudes, those are all harmless. They're the position of the old Tysons office and the geographical center of McLean.